cypress/cypress/views/submariner/actions/submariner_actions.js

```diff
@@ -18,31 +18,40 @@ export const submarinerClusterSetMethods = {
 
     // The function adds the AWS an GCP clusters to the cluster set.
     // start position: from the 'Cluster set' list.
-    manageClusterSet: (clusterSetName) => {
+    manageClusterSet: (clustersList, clusterSetName) => {
+        cy.log(clustersList)
         cy.get('[data-label="Name"]').contains(clusterSetName).click()
         cy.get('#clusters').contains('Go to Managed clusters').click()
-        cy.get('.pf-c-empty-state__primary > .pf-c-button').click()
+        cy.wait(500)
```

Comment:
Better to avoid such a "stupid" wait.
Instead to check for condition state we expect to get.